timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchIntegrationTestUtilities.java

```diff
@@ -174,6 +175,16 @@ private static Schema createSchema() {
         };
         schema.addTableDefinition(TABLE_2, tableWithoutCaching);
 
+        TableDefinition tableWithRowConflictHandler = new TableDefinition() {
+            {
+                rowName();
+                rowComponent("key", ValueType.BLOB);
+                noColumns();
+                conflictHandler(ConflictHandler.SERIALIZABLE);
+            }
+        };
+        schema.addTableDefinition(SERIALIZABLE_TABLE, tableWithRowConflictHandler);
+
```

Comment:
I want to make sure that metadata also works with a table that uses row-level locks (and SERIALIZABLE conflict handler) as that will be the most common use case. Since AtlasDB caching does not work out-of-the-box with row-level locks, we have to register the watch ourselves